src/library_fs.js

```diff
@@ -895,6 +896,9 @@ FS.staticInit();
       FS.readdirNode(FS.lookupPath(path, { follow: true }).node);
     },
     readdirNode(node) {
+      if (!node.parent) {
+        return [];
+      }
```

Comment:
Any idea what these lines were supposed to be doing?

What does not having a parent mean?  Is that only true for the root directory?  Why would we want that to mean return the empty list?

Reply:
Well, if we've unlinked a directory we want to return empty list or raise ENOENT. By contrast, if the directory exists but is empty we return `[".", ".."]`. So we need to label the directory node as unlinked somehow when we unlink them. I can add a comment here saying that.